openassessment/xblock/openassessmentblock.py

```diff
@@ -119,9 +119,15 @@ class OpenAssessmentBlock(
     )
 
     allow_file_upload = Boolean(
-        default=False,
+        default=None,
+        scope=Scope.content,
+        help="Do not use. For backwards compatibility only."
```

Comment:
It's fine as it. Please ignore this as we are not apply i18n on rest of the fields.